src/test/java/picard/fingerprint/CheckFingerprintTest.java

```diff
@@ -125,6 +125,20 @@ public void testMismatchingSamples2() {
         Assert.assertEquals(runPicardCommandLine(args), 1);
     }
 
+    @Test
```

Comment:
Shouldn't you have tests to validate that you get the two different exit codes for the two different code paths you have added?